css/index.css

```diff
@@ -0,0 +1,211 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  box-sizing: border-box;
+  padding: 0%;
+  margin: 0;
+}
+/* styling the header */
+.store {
+  color: #e0633a;
+}
+header {
+  background-color: #ebe4e4;
+}
+/* styling h1 */
+h1 {
+  margin-bottom: 50px;
+  color: #de6e49;
+}
+
+/* styling the form */
+form {
+  display: flex;
+  padding-left: 40px;
+  width: 45%;
+  margin: 3% 5%;
+  flex-direction: column;
+  font-size: 15px;
+  height: auto;
+}
+.order-form {
+  display: flex;
+  flex-direction: row;
+}
+
+/* styling the input */
+input,
+#city {
+  padding: 15px;
+  margin-bottom: 20px;
+}
+
+/* styling the label */
+label {
+  margin-bottom: 5px;
+}
+
+/* putting the first and last name in a row */
+.full-name {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* putting first and last name in a column */
+.name-details {
+  display: flex;
+  flex-direction: column;
+  width: 48%;
+}
+
+/* styling address */
+.address-one {
+  display: flex;
+  flex-direction: column;
+}
+.address {
+  display: flex;
+  flex-direction: column;
+  width: 48%;
+}
+
+/* styling city and postcode */
+.city-postcode {
+  display: flex;
+  justify-content: space-between;
+}
+
+#city {
+  border: 0.5px solid black;
+}
+
+/* styling the button */
+button {
+  border-radius: 4px;
+  background-color: #e0633a;
+  cursor: pointer;
+  padding: 15px;
+  width: 120px;
+  color: white;
+  border-style: none;
+  font-size: 11px;
```

Comment:
I would put the `font-size` of buttons bigger (e.g. `13px`) to help users with bad eye sight to be able to read them.